ci/environment-dev-3.6.yml

```diff
@@ -11,12 +11,13 @@ dependencies:
   # IDE
   - jupyterlab
   # Input/Output
-  - netcdf4
+  - netcdf4=1.5.1.1
```

Comment:
maybe >= is better?

Reply:
@ahuang11 Newer versions of netcdf4 break some xarray functionality (at least for me) so for now I have set netcdf4 to the newest version that works. See: https://github.com/pydata/xarray/issues/3580

Reply:
FYI @kpegion, these are the kinds of things worth adding an explicit test for. I'm trying to get better about promoting that when a bug comes up you fix it and add a test to ensure it doesn't break again.

A good example is here: https://github.com/bradyrx/climpred/pull/276